src/api/api_client.rs

```diff
@@ -30,58 +30,6 @@ use sp_version::RuntimeVersion;
 /// Api to talk with substrate-nodes
 ///
 /// It is generic over the `Request` trait, so you can use any rpc-backend you like.
-///
-/// # Custom Client Example
```

Comment:
Removed the custom client example, because it was not easily possible to make it compile for both sync and async features and is already described in the README https://github.com/scs/substrate-api-client#rpc-client